be/src/olap/txn_manager.cpp

```diff
@@ -263,6 +266,10 @@ OLAPStatus TxnManager::publish_txn(OlapMeta* meta, TPartitionId partition_id, TT
     }
 }
 
+// txn could be rollbacked if it does not have related rowset
+// if the txn has related rowset then could not rollback it, because it 
+// may be committed in another thread and our current thread meets erros when writing to data file
+// be has to wait for fe call clear txn api
```

Comment:
In what scenario, one thread want to rollback and another thread commit it?

Reply:
In stream load scenario, if coordinator be send two ingest commands, there maybe two running thread to load the rowset.